src/Symfony/Component/Config/Definition/Builder/NormalizationBuilder.php

```diff
@@ -45,7 +45,7 @@ public function remap(string $key, string $plural = null): static
     /**
      * Registers a closure to run before the normalization or an expression builder to build it if null is provided.
      *
-     * @return ExprBuilder|$this
+     * @return $this|ExprBuilder
```

Comment:
I'd force "$this" / self / etc at right :|

Reply:
Currently, Fixer does not support such logic.
Feel free to introduce it via [raising PR](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/compare)